internal/controller/dispatcher.go

```diff
@@ -257,3 +265,41 @@ func (r *Dispatcher) dispatch(ctx context.Context) (ctrl.Result, error) {
 
 	return ctrl.Result{RequeueAfter: dispatchDelay}, nil
 }
+
+// Calculate resource demands of appWrappers that have been dispatched but haven't
+// passed through ResourceQuota admission controller yet (approximated by resources not created yet)
+func (r *Dispatcher) getUnadmittedAppWrappersWeights(ctx context.Context) (map[string]*WeightsPair, error) {
```

Comment:
Maybe rename to `getUnadmittedAppWrapperPodsWeights`?

Reply:
done